### DISCUSSION TITLE
Is this a problem? Ensure that you adopt SDKs which support app set ID

### DISCUSSION THREAD

**Discussion by jj449** [admob-plus/admob-plus#527] [Q&A]
I got mail from admob : 

Ensure that you adopt SDKs which support app set ID
--
We noticed that you have not adopted app set ID for your apps.
Last July, Google Play announced advertising ID policy changes to help boost security and privacy. These changes have started to come into effect from April 1, 2022.
App set ID offers a privacy-friendly way to correlate usage or actions across a set of apps owned by the same organization when the advertising ID is unavailable. To support reporting and fraud prevention of opt-ed out users, Google AdMob recommends taking 20.5.0+ GMA SDK or later which support app set ID. After which, no further action is required.

I wonder if this will be a problem ?

**Comment by Calsy** [comment]
Received multiple emails from Google about the need to support App Set ID.  Surprised not much interest in this issue as ad serving will break on Android once they start forcing apps to adopt these new policies.

**Reply by Carlosps** [reply]
Let's fund this feature so that @ratson or anyone else could create a pull request to help us with this?

**Reply by jj449** [reply]
good idea ! but I don't know how to do it ...where should we start the funding for @ratson?

**Reply by Carlosps** [reply]
There is this https://github.com/admob-plus/admob-plus/discussions/314. I guess we should include this feature on this list.

**Reply by Calsy** [reply]
Sounds like a good idea. Will have to try getting in contact with @ratson first.

**Reply by r2uan** [reply]
> Let's fund this feature so that @ratson or anyone else could create a pull request to help us with this?

I started sponsoring @ratson on github a few months ago, might be worth doing! Hopefully he'll see this soon and we get some updates
